shell/config/router.js

```diff
@@ -89,7 +89,7 @@ export const routerOptions = {
           name:      'account-create-key'
         },
         {
-          path:      'auth',
```

Comment:
This configuration wasn't actually being used correctly. It was falling back on `/c/:cluster/:product`

![auth](https://github.com/rancher/dashboard/assets/55104481/33b64ca5-e5b0-4c8d-a38f-77e2ba339de2)